lesson16/main.js

```diff
@@ -0,0 +1,119 @@
+"use strict";
+
+const jsonUrl = "https://jsondata.okiba.me/v1/json/yFXrE210903000829";
+const newsWrapper = document.getElementById("js-news");
+
+const createElementWithClassName = (element, name) => {
+  const variableName = document.createElement(element);
+  variableName.classList.add(name);
+  return variableName;
+};
+
+//comment-icon
+const commentIcon = createElementWithClassName("span", "comment-icon");
+
+//new-label
+const newLabel = createElementWithClassName("span", "new-label");
+newLabel.textContent = "NEW!";
+
+//news-img
+const imgWrapper = createElementWithClassName("div", "news__img");
+
+
+//tab
+const createNewTab = data => {
+  const tabUl = document.querySelector(".news__tabs");
+  const tabFragment = document.createDocumentFragment();
+
+  data.forEach((key) => {
+    const tabLi = createElementWithClassName("li", "tab");
+    tabLi.textContent = key.category;
+    tabLi.dataset.tab = key.id;
+    if (key.selected === true) {
+      tabLi.classList.add("is-active");
+    }
+
+    tabLi.addEventListener("click", tabSwitch);
+    tabFragment.appendChild(tabLi);
+  });
+
+  tabUl.appendChild(tabFragment);
+};
+
+//news
+const createNewContent = data => {
+  const contentFragment = document.createDocumentFragment();
+  
+  data.forEach( key => {
+    const newsContent = createElementWithClassName("div", "news__content");
+    const contentUl = createElementWithClassName("ul", "news__lists");
```

Comment:
IMO
```suggestion
    const contentUl = createElementWithClassName("ul", "js-news__lists");
```
With this class name, it seems difficult to understand that it is controlled by js when applying style with CSS.

Reply:
I understand, I don't have this idea! Thank you so much.